__tests__/utils.test.js

```diff
@@ -102,3 +103,21 @@ describe("formatComments", () => {
     expect(formattedComments[0].created_at).toEqual(new Date(timestamp));
   });
 });
+
+describe("POST/ formatComment", () => {
```

Comment:
this is a bit unclear as it suggests the util is part of the api request, i think just removing the POST/ from the description would suffice, maybe call the function formatPostedComment if you want to give it more semantic meaning